config/environments/test.rb

```diff
@@ -1,6 +1,10 @@
 # frozen_string_literal: true
 
 Rails.application.configure do
+  # By default, when set to `nil`, strategy loading errors are suppressed in test
+  # mode. Set to `true` to always raise errors, or `false` to always warn.
+  config.flipflop.raise_strategy_errors = nil
```

Comment:
I think I would probably set this to `true` in the test environment, so we can head off any strategy issues before they hit production

Reply:
@sandbergja  - I'm not seeing this change at all in `config/environments/test.rb`? I was expecting to see `config.flipflop.raise_strategy_errors = true` in that file?

Reply:
Ah, I made a bad assumption that it would default to `true` if I left it out.  Added that back in.